structure/0006-governance.md

```diff
@@ -71,42 +72,45 @@ The RFC proposing the new team should contain the following major sections:
+
+Individuals can be added to a Supporting Team through informal consensus within the relevant teams, without the need for a proper vote to take place,
+as members of Supporting Teams don't have the sensitive access usually granted to regular team members.
 
 ## The Administrative Board
 
 The administrative board is a group of organization members responsible for tasks that are not assigned to another team.
 It is made up of at least three members to prevent deadlocks in the decision-making process.
 Members of the administrative board may be members of other teams.
 
-If it is decided that more administrative board members are required, either due to a current member stepping down, or due to it being decided that the admin board needs to be expanded, an election must take place.
+If it is decided that more administrative board members are required, either due to a current member stepping down, or due to it being decided through informal concensus that the admin board needs to be expanded, an election must take place.
 
-### Admin board Voting Process
+### Admin Board Election Process
 
- Due to the fact that multiple candidates may be nominated for the admin position when a vote is held, the fact that there may be multiple spots to fill on the board, and the fact that the decision affects the entire organization, the standard voting procedure cannot be used. Instead, the following procedure will be employed:
+Due to the fact that multiple candidates may be nominated for the admin position when a vote is held, the fact that there may be multiple spots to fill on the board, and the fact that the decision affects the entire organization, the standard voting procedure cannot be used. Instead, the following procedure will be employed:
 
 Before the vote begins, a group of people, henceforth referred to as the Election Oversight Board or EOB, must be selected to oversee the voting process. They will be responsible for receiving candidate nominations, concerns about nominated candidates from other organization members, and hosting the election. In normal circumstances, the EOB consists of the existing members of the admin board, but, if this is not possible for any reason, the EOB will instead consist of members with the Community Manager role in the Community Team. The initial EOB may add any members that they feel would be a valuable addition to the board, with a simple majority vote.
 
 The vote is conducted using [Single Transferrable Vote](https://en.wikipedia.org/wiki/Single_transferable_vote) (STV), or [Instant Runoff voting](https://en.wikipedia.org/wiki/Instant-runoff_voting) (IRV) in instances where there is only one space on the admin board to be filled, to ensure a minimum amount of "wasted votes". When using Single Transferrable vote, the [Droop Quota](https://en.wikipedia.org/wiki/Droop_quota) must be used to calculate the amount of votes required for a candidate to win a position on the admin board.
```

Comment:
What happens when the amount of candidates is equal to the amount of open positions being considered in an upcoming election? Do they automatically win those positions and become members of the Administrative Board? 

I'd think there should be a form of confirmatory vote in this situation, so the electorate (the members of the organization) have a chance to discuss the administrative board members-elect and ultimately confirm or deny them their positions, rather than automatically conferring those positions as uncontested.